### ISSUE TITLE
Provide a REST API for the Job Management

### ISSUE THREAD

**Issue by ansgarS** [ansgarS/rate-my-bistro-crawler#4]
Create a small REST API, that can handle the following requests:

```
POST /jobs/<dateRange>
RETURNS JobId

GET /jobs/<JobId>
RETURNS JobInformaiton

GET /jobs
RETURNS List<JobId>
```

**Comment by ansgarS** [comment]
Use https://github.com/go-swagger/go-swagger

**Comment by ansgarS** [comment]
?from=xxxx&to=xxxx

**Comment by RouHim** [comment]
solved by #14
